plugins/thumbnails/server/worker.py

```diff
@@ -162,14 +165,16 @@ def attachThumbnail(file, thumbnail, attachToType, attachToId, width, height):
     return ModelImporter.model('file').save(thumbnail)
 
 
-def getImage(extension, data):
+def _getImage(mimeType, extension, data):
     """
     Check extension of image and opens it.
 
     :param extension: The extension of the image that needs to be opened.
     :param data: The image file stream.
     """
-    if 'dcm' in extension:
+    if (extension and extension[-1] == 'dcm') or mimeType == 'application/dicom':
+        if not numpy:
+            raise Exception('Missing dependancy: numpy.')
```

Comment:
Minor nit, but let's change this error message to:

    raise Exception('Could not create DICOM thumbnail: numpy not installed.')